src/ai/backend/manager/api/auth.py

```diff
@@ -574,6 +574,18 @@ async def wrapped(request, *args, **kwargs):
     return wrapped
 
 
+def auth_required_for_method(method):
```

Comment:
We may also need to create an `admin_required_for_method`, just like with `admin_required`.